src/main/steps/timed-out/get.test.ts

```diff
@@ -15,18 +13,10 @@ describe('TimedOutGetController', () => {
     const res = mockResponse();
     await controller.get(req, res);
     const language = SupportedLanguages.En;
-    const isDivorce = true;
-    const userCase = req.session.userCase;
 
-    expect(req.session.destroy).toHaveBeenCalled();
+    expect(req.session.destroy).toBeCalled();
```

Comment:
this needs to remain as toHaveBeenCalled as per a new eslint upgrade - https://github.com/hmcts/nfdiv-frontend/pull/2561/commits/d942698883572a6d4667e5bd93824ffa0ed809ff

Reply:
done